src/ingameop.cpp

```diff
@@ -572,9 +570,8 @@ static bool startIGOptionsMenu()
 	addIGTextButton(INTINGAMEOP_MOUSEOPTIONS, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_OP_W, _("Mouse Options"), OPALIGN);
 	row++;
 
-	bool s = bMultiPlayer && NetPlay.bComms != 0;
 	// Key map editor does not allow editing for true multiplayer
-	addIGTextButton(INTINGAMEOP_KEYMAP, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_OP_W, s? _("View Key Mappings") : _("Key Mappings"), OPALIGN);
+	addIGTextButton(INTINGAMEOP_KEYMAP, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_OP_W, (bMultiPlayer && NetPlay.bComms) ? _("View Key Mappings") : _("Key Mappings"), OPALIGN);
```

Comment:
This gets a bit complicated. Can we put `(bMultiPlayer && NetPlay.bComms) ? _("View Key Mappings") : _("Key Mappings")` into it's own variable like `keymapButtonText`?